test/functional/vimscript/system_spec.lua

```diff
@@ -558,9 +558,13 @@ end)
 describe('shell :!', function()
   before_each(clear)
 
-  it(':{range}! with powershell filter/redirect #16271 #19250', function()
+  it(':{range}! with powershell using "commands" filter/redirect #16271 #19250', function()
+    if not n.has_powershell() then
+      pending('powershell not found', function() end)
+      return
```

Comment:
Let's just `return`. It's noisy to see lots of "skipped" tests for CI envs that don't have powershell. Same for the other cases below.
```suggestion
      return
```